File: Equipments.Domain/Entities/EquipmentModel.cs
namespace Equipments.Domain.Entities
{
    public class EquipmentModel : Entity
    {
        public string Name { get; private set; }

        public EquipmentModel(string name)
        {
            Name = name;
        }

        public void EditName(string name) => Name = name;
    }
}

File: equipments/src/main/java/com/br/raf/equipments/entities/EquipmentModel.java
package com.br.raf.equipments.entities;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.util.UUID;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "equipment_model")
public class EquipmentModel {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private UUID id;
    private String name;
}
